jest.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   testEnvironment: 'jest-environment-jsdom',
   testRegex: 'src/.+\\.(test|spec)\\.[jt]sx?$',
-  setupFilesAfterEnv: ['./jest.setup.js'],
+  setupFilesAfterEnv: ['./jest.setup.spec.js'],
```

Comment:
Why was the name change required?

Reply:
The file contains jest specifics so it needs to be recognised as a test file (beforeEach and jest.spyOn)